content/prerequisites/awscli/index.md

```diff
@@ -1,28 +0,0 @@
----
```

Comment:
I assume we are discontinuing (or refactoring) this page in favor of the AWS Cloud9-based approach.  Have we checked if this page is being referenced in other labs, so to avoid broken links?  It seems you are moving this page to another section: if this is correct, can you look and update links, if any, accordingly?

Reply:
we do not discontinuing, C9 is a addition to local development. the awscli was moved there as C9 comes with the tool pre-installed...looking for links and will update accordingly

Reply:
found one..updated